addons/l10n_mx/data/account_report_diot.xml

```diff
@@ -149,6 +149,11 @@
                 <field name="groupby">partner_id, id</field>
                 <field name="hierarchy_level">0</field>
                 <field name="expression_ids">
+                    <record id="tax_report_mx_diot_paid_8_n_base_tag" model="account.report.expression">
```

Comment:
Instead of creating new expressions, you could just reuse the old ones and just change their formulas and/or label. 

Then, you also don't necessarily need to change the label either. Whether you choose to change the label or not is mainly a functional decision:
- reuse the old labels -> the amounts will stay the same until the user upgrades the module
- create new labels -> the amounts will show zero until the user upgrades the module